sources/packages/web/src/services/http/EducationProgramApi.ts

```diff
@@ -15,12 +15,10 @@ import HttpBaseClient from "./common/HttpBaseClient";
 import { addSortOptions } from "@/helpers";
 
 export class EducationProgramApi extends HttpBaseClient {
-  public async getProgram(programId: number): Promise<any> {
+  public async getProgram(programId: number, locationId: number): Promise<any> {
```

Comment:
I know, its not a part of your PR. since you are already working on this function, can you create a return interface instead of `any`